src/main/java/uk/gov/pay/adminusers/service/NotificationService.java

```diff
@@ -78,45 +75,43 @@ NotifyDirectDebitConfiguration getNotifyDirectDebitConfiguration() {
-        personalisation.put("username", sender);
-        personalisation.put("link", inviteUrl);
+        Map<String, String> personalisation = Map.of(
+                "username", sender,
+                "link", inviteUrl);
         return sendEmailAsync(CARD, inviteEmailTemplateId, email, personalisation);
     }
 
     CompletableFuture<String> sendServiceInviteEmail(String email, String inviteUrl) {
-        HashMap<String, String> personalisation = newHashMap();
-        personalisation.put("name", email);
-        personalisation.put("link", inviteUrl);
+        Map<String, String> personalisation = Map.of(
+                "name", email,
+                "link", inviteUrl);
         return sendEmailAsync(CARD, notifyConfiguration.getInviteServiceEmailTemplateId(), email, personalisation);
     }
 
     CompletableFuture<String> sendForgottenPasswordEmail(String email, String forgottenPasswordUrl) {
-        HashMap<String, String> personalisation = newHashMap();
-        personalisation.put("code", forgottenPasswordUrl);
+        Map<String, String> personalisation = Map.of("code", forgottenPasswordUrl);
         return sendEmailAsync(CARD, forgottenPasswordEmailTemplateId, email, personalisation);
     }
 
     CompletionStage<String> sendServiceInviteUserExistsEmail(String email, String signInLink, String forgottenPasswordLink, String feedbackLink) {
-        HashMap<String, String> personalisation = newHashMap();
-        personalisation.put("signin_link", signInLink);
-        personalisation.put("forgotten_password_link", forgottenPasswordLink);
-        personalisation.put("feedback_link", feedbackLink);
+        Map<String, String> personalisation = Map.of(
+                "signin_link", signInLink,
+                "forgotten_password_link", forgottenPasswordLink,
+                "feedback_link", feedbackLink);
         return sendEmailAsync(CARD, notifyConfiguration.getInviteServiceUserExistsEmailTemplateId(), email, personalisation);
     }
 
     CompletableFuture<String> sendServiceInviteUserDisabledEmail(String email, String supportUrl) {
-        HashMap<String, String> personalisation = newHashMap();
-        personalisation.put("feedback_link", supportUrl);
+        Map<String, String> personalisation = Map.of("feedback_link", supportUrl);
         return sendEmailAsync(CARD, notifyConfiguration.getInviteServiceUserDisabledEmailTemplateId(), email, personalisation);
     }
 
     CompletableFuture<String> sendInviteExistingUserEmail(String sender, String email, String inviteUrl, String serviceName) {
         String collaborateServiceNamePart, joinServiceNamePart = "";
-        HashMap<String, String> personalisation = newHashMap();
-
-        personalisation.put("username", sender);
-        personalisation.put("link", inviteUrl);
+        HashMap<String, String> personalisation = new HashMap<>() {{
+            put("username", sender);
+            put("link", inviteUrl);
+        }};
```

Comment:
Can just use `Map.of(…)` here.

Reply:
True, I didn't think about moving the declaration further down the method to where we actually know what we want to put into it.